### PR TITLE
fix: UI for curly braces in property pane suggestions

### PR DESCRIPTION
**Pull request by ankitakinger**
## Description

UI for curly braces in property pane suggestions

Fixes # (issue number)

#### Type of change
- Chore (housekeeping or task changes that don't impact user perception)

#### How Has This Been Tested?
- [x] Manual
- [ ] Jest
- [ ] Cypress

#### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Test-plan-implementation#speedbreaker-features-to-consider-for-every-change) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans/_edit#areas-of-interest)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [ ] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

File: app/client/src/pages/Editor/PropertyPane/PropertyControl.tsx [hunk 1]
```diff
@@ -764,17 +764,13 @@ const PropertyControl = memo((props: Props) => {
...
-                    isDisabled={isToggleDisabled}
-                    isSelected={isDynamic}
-                    onClick={() =>
-                      toggleDynamicProperty(propertyName, isDynamic)
-                    }
-                    size="sm"
-                  />
-                </div>
+                <ToggleButton
+                  icon="js-toggle-v2"
```
Comment (yaldram) [inline app/client/src/pages/Editor/PropertyPane/PropertyControl.tsx:768]: Please add this to the JS Toggle Button, it is used for testing - 
```js
className={classNames("t--js-toggle", {
     "is-active": isDynamic,
 })}
````

File: app/client/src/components/propertyControls/MenuButtonDynamicItemsControl.tsx [hunk 2]
```diff
@@ -25,12 +25,13 @@ const PromptMessage = styled.span`
...
 const CurlyBraces = styled.span`
-  color: ${(props) => props.theme.colors.codeMirror.background.hoverState};
-  background-color: #575757;
+  color: var(--ads-v2-color-fg);
+  background-color: var(--ads-v2-color-bg-muted);
   border-radius: 2px;
   padding: 2px;
   margin: 0px 2px;
   font-size: 10px;
+  font-weight: bold;
```
Comment (yaldram) [inline app/client/src/components/propertyControls/MenuButtonDynamicItemsControl.tsx:34]: Please use this variable for bold fonts - `--ads-v2-font-weight-bold`

**Review by yaldram — CHANGES_REQUESTED** [review]

**Review by yaldram — APPROVED** [review]

**Review by rohitagarwal88 — APPROVED** [review]

**Pull request merged**